frontend/src/components/explorer/interactionPartners/DetailsPage.vue

```diff
@@ -537,7 +571,11 @@ export default {
               color = colorToRGBArray(this.defaultMetaboliteColor);
             }
           }
-          colors[node.id] = color;
+          if (node.id === this.clickedElmId) {
+            colors[node.id] = colorToRGBArray(NODE_SELECT_COLOR);
+          } else {
+            colors[node.id] = color;
+          }
```

Comment:
Just curious, why is this included in the PR as it does not seem to be related to sizing?

Reply:
Good question! In `handleWindowResize`, we need to call `await this.applyColorsAndRenderNetwork();`. If there is a selected node and we don't have this then the node would lose the red color. It's an edge case as I don't think users would be resizing the window a lot when using this page but I didn't want to introduce any new buggy behavior with this PR so it's included.

Reply:
Ah, makes sense! Nice catch